### ISSUE TITLE
Actor section lacking usage example and important properties

### ISSUE THREAD

**Issue by schnittchen** [ponylang/pony-tutorial#469]
Currently there is no code example of creating and using an actor.

I'm currently reading through one of the pony papers, and, "All actors in Pony are visible to other actors with capability tag but see themselves as ref." is not reflected in the section.

**Comment by ergl** [comment]
For the "see themselves as `ref`", we could probably reuse the material for the ["Accessing an Actor with Arbitrary Transactions"](https://patterns.ponylang.io/async/access.html#solution), although that requires talking about capabilities, which are not introduced until later in the tutorial.

**Comment by jemc** [comment]
I agree that linking to that Pony Pattern would be helpful for a deep dive into this concept, which we can cover more briefly and succinctly in the tutorial.

**Comment by adrianboyko** [comment]
> I'm currently reading through one of the pony papers, and, "All actors in Pony are visible to other actors with capability tag but see themselves as ref." is not reflected in the section.

The sections of the tutorial before "Reference Capabilities" seem to intentionally present text and examples that work without explicit capabilities being specified. I think that's a good idea because one can get a long way into learning Pony before explicit caps become necessary. Because "Actors" is one of the sections before "Reference Capabilities" I think it's correct to not talk about `tag` or `ref` here.
